session/bootstrap.go

```diff
@@ -670,6 +675,13 @@ func upgradeToVer24(s Session) {
 	writeSystemTZ(s)
 }
 
+// upgradeToVer25 updates tidb_max_chunk_size to new low bound value 32 if previous value is small than 32.
+func upgradeToVer25(s Session) {
+	sql := fmt.Sprintf("UPDATE HIGH_PRIORITY %s.%s SET VARIABLE_VALUE = '32' WHERE VARIABLE_NAME = '%s' AND VARIABLE_VALUE < 32",
```

Comment:
maybe it's better to use "DefTiDBMaxChunkSize" to replace the magic number "32"?